docs/pages/reference/helm-reference/teleport-kube-agent.mdx

```diff
@@ -106,6 +106,31 @@ This parameter is not mandatory to preserve backwards compatibility with older c
 
 If you specify a role here, you may also need to specify some other settings which are detailed in this reference.
 
+## `enterprise`
+
+| Type     | Default value |
+|----------|---------------|
+| `bool`   | `false`       |
+
+When `enterprise` is set to `true`, the container image used for Teleport agent
+pods run by the `teleport-kube-agent` chart will be the enterprise version.
+This should be set to `true` for connecting to Teleport Cloud and self-hosted
```

Comment:
@stevenGravy shouldn't this specify "Cloud Enterprise", or should those on the Cloud Team plan also use the enterprise binaries?

Reply:
TEam uses enterprise binaries too.